scripts/ganache.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+helpFunction() {
+	echo "Usage: $0 [start|stop] [background]"
+	exit 1 # Exit script after printing help
+}
+
+start() {
+	source scripts/env.sh
+	WEB3_SYSTEM_TEST_BACKEND="ganache"
+
+	echo "Starting gancahe ..."
+	echo "npx ganache ethereum -m \"$WEB3_SYSTEM_TEST_PASSWORD\" -a 5 -p $WEB3_SYSTEM_TEST_HTTP_PORT &"
+	npx ganache ethereum -m "$WEB3_SYSTEM_TEST_PASSWORD" -a 5 -p $WEB3_SYSTEM_TEST_HTTP_PORT >/dev/null &
+
+	echo "Waiting for ganache..."
+	npx wait-port "$WEB3_SYSTEM_TEST_HTTP_PORT"
+	sleep 2
```

Comment:
Is this `sleep` necessary? Shouldn't `wait-port` be enough?

Reply:
Sometimes we have to wait for few ms to listen to events after sending a transaction.